workers/unity/Packages/com.improbable.gdk.buildsystem/Configuration/SpatialOSBuildConfiguration.cs

```diff
@@ -47,26 +47,22 @@ private void ResetToDefault()
         {
             WorkerBuildConfigurations = new List<WorkerBuildConfiguration>
             {
+                new WorkerBuildConfiguration
                 {
-                    new WorkerBuildConfiguration
-                    {
-                        WorkerType = "UnityClient",
-                        LocalBuildConfig = new BuildEnvironmentConfig { BuildOptions = BuildOptions.Development },
-                    }
+                    WorkerType = "UnityClient",
+                    LocalBuildConfig = new BuildEnvironmentConfig { BuildOptions = BuildOptions.Development },
                 },
+                new WorkerBuildConfiguration
                 {
-                    new WorkerBuildConfiguration
+                    WorkerType = "UnityGameLogic",
+                    LocalBuildConfig =
+                        new BuildEnvironmentConfig { BuildOptions = BuildOptions.EnableHeadlessMode },
+                    CloudBuildConfig = new BuildEnvironmentConfig
                     {
-                        WorkerType = "UnityGameLogic",
-                        LocalBuildConfig =
-                            new BuildEnvironmentConfig { BuildOptions = BuildOptions.EnableHeadlessMode },
-                        CloudBuildConfig = new BuildEnvironmentConfig
-                        {
-                            BuildPlatforms = SpatialBuildPlatforms.Linux,
-                            BuildOptions = BuildOptions.EnableHeadlessMode
-                        }
+                        BuildPlatforms = SpatialBuildPlatforms.Linux,
+                        BuildOptions = BuildOptions.EnableHeadlessMode
                     }
-                }
+                },
```

Comment:
if we start doing trailing commas, please lets do them everywhere 👍

Reply:
I can make a pass, but would prefer in a separate PR.
But definitely want trailing commas ;)

Reply:
I thought our convention was trailing commas

Reply:
but it hasn't been done everywhere afaik